profiles/package.mask

```diff
@@ -29,6 +29,11 @@
 
 #--- END OF EXAMPLES ---
 
+# Jimi Huotari <chiitoo@gentoo.org> (21 Jun 2019)
+# Deprecated package with no consumers left.
+# Removal in 30 days.
```

Comment:
However, this PR is removing it already. Last-rites are correct, removal can only be done in 30 days.

Reply:
I was looking at 'lxqt-common' for reference [1][2], as it was similarly removed, and from those commits I figured, since the version would be dropped, I would drop it the same way, but leave the metadata around for the thirty days.  Would be sort of weird for it to stay hanging around after all its buddies are gone after all.

Did I misunderstand some part(s) still?

1. https://gitweb.gentoo.org/repo/gentoo.git/commit/lxqt-base/lxqt-common?id=06a16e24a6509e7dbf304b904e2adc61546c783a
2. https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=2eb558c301408c9ae890f5b0bb6ab898f097a932

Reply:
Well, you've just discovered a mistake of mine. ;)

Reply:
Heh.  So, the correct approach would indeed be to leave the ebuild be, until that 30 days has passed?

Reply:
yes, even if no other ebuild is using it anymore

Reply:
Makes sense, too.

Should be done with the latest push.